app/models/terms_of_use_agreement.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class TermsOfUseAgreement < ApplicationRecord
+  belongs_to :user_account
+
+  validates :agreement_version, :response, presence: true
+  enum response: { declined: 0, accepted: 1 }
```

Comment:
Testing passes however when a response is submitted the created_at and updated_at are the same time every time I submit a new response. In future versions of this we will need an audit of all submissions to show when a person accepted and denied every time they do.

Reply:
It will create a new record for every response so a user could potentially have multiple records. The latest record will be used as their current response